src/components/message/index.tsx

```diff
@@ -195,9 +194,18 @@ export class Message extends React.Component<Properties, State> {
     );
   }
 
+  // only render the name in the top message of a group
```

Comment:
Is this comment still valid?

Reply:
no, i think i'll put this back where it was before. Earlier i had extract this to a function